packages/ses/src/error/unhandled-rejection.js

```diff
@@ -63,7 +63,7 @@ export const makeRejectionHandlers = reportReason => {
    * @param {ReasonId} heldReasonId
    */
   const finalizeDroppedPromise = heldReasonId => {
-    if (mapHas(idToReason)) {
+    if (mapHas(idToReason, heldReasonId)) {
```

Comment:
That's quite a bug!

Why did the old code seem to work?

Reply:
Old code probably didn't when the promise would get collected. `map.has(undefined)` was always `false` so we never reported then. However we also had a report path on exit.